www/index.html

```diff
@@ -0,0 +1,439 @@
+                </div>
+            </div>
+        </div>
+
+
+        <hr class="featurette-divider">
+        <div id="download" class="container pad">
+            <h2 class="text-center">Download Instructions</h2>
+            <div class="row">
+                <div class="col-lg-6">
+                    <h3 class="pad">Download</h3>
+                    <form method="get" action="../build/webgazer.js">
+                        <button class="btn btn-primary btn-lg" type="submit"><span class="glyphicon glyphicon-cloud-download" aria-hidden="true"></span> Download WebGazer.js</button>
+                    </form>
+                </div>
+                <div class="col-lg-6">
+                    <h3 class="pad">Create from Source</h3>
+                    The GitHub repository can be found <a href="https://github.com/brownhci/WebGazer.git">here</a>.
+                    <pre><code class="bash"># Ensure NodeJS is downloaded: https://nodejs.org/en/download/
+# Install grunt-cli if not installed (this may require you to use `sudo` or run the console as Administrator)
+npm install -g grunt-cli
+git clone https://github.com/brownhci/WebGazer.git
+cd WebGazer
+npm install
+# Run grunt to build the webgazer.js file in the build directory
+grunt</code></pre>
+                </div>
+            </div>
+        </div>
+
+
+        <hr class="featurette-divider">
+        <div id="examples" class="container pad">
+            <h2 class="text-center pad">Examples</h2>
+            <div class="row">              
+                <div class="col-lg-4">
+                    <img src="media/empty.png" alt="Empty Webpage Demo" width="300" height="250">
+                    <h4 class="pad">WebGazer.js on an Empty Webpage with calibration</h4>
+                    <p>See how easy it is to integrate WebGazer.js on any webpage. Just with a few clicks you will get real-time predictions.  The only thing you need to do is click on these calibration points within the screen while looking at the cursor. Both clicks and cursor movements make the predictions more accurate. The video and the visualized gaze prediction are only shown in debugging mode.
+                        <form method="get" action="https://webgazer.cs.brown.edu/demo.html">
+                        <button class="btn btn-primary" type="submit">Try Live!</button>
+                    </form>
+                    </p>
+                </div>
+                <div class="col-lg-4">
+                    <img src="media/collision.png" alt="Collision demo" width="300" height="250">
+                    <h4 class="pad">Ball Collision Game</h4>
+                    <p>Move the orange ball with your eyes and create collisions with the blue balls. Train WebGazer.js by clicking in various locations within the screen, while looking at your cursor.                     
+                        <form method="get" action="https://webgazer.cs.brown.edu/collision.html">
```

Comment:
Do these paths have to be hard-coded? It's just someone testing the code locally then gets redirected to the website instead of their local files.

Reply:
I don't believe so.